frontend/components/commons/sidebar-feedback-task/SidebarFeedbackTask.component.vue

```diff
@@ -53,18 +53,19 @@ export default {
   },
   created() {
     this.sidebarItems = {
-      mode: {
-        buttonType: "non-expandable",
-        buttons: [
-          {
-            id: "annotate",
-            tooltip: "Hand labeling",
-            icon: "hand-labeling",
-            action: "change-view-mode",
-            relatedMetrics: ["progress", "stats"],
-          },
-        ],
-      },
+      // NOTE: HIDDEN FOR MVP
```

Comment:
1/ install the plugin Comment Anchor in VSC reach easily the comment with specific keyword
2/ replace NOTE by TODO